api/app/controllers/review.controller.ts

```diff
@@ -134,6 +134,7 @@ export async function findAll(req: Request, res: Response) {
 // Find a single review with an id
 export async function findOne(req: Request, res: Response) {
   const id = parseInt(req.params.id);
+  const userId = req.body.userId;
```

Comment:
Since there is a req.body here, there will be a modification on the FE making a request on this controller route?

Reply:
No. The middleware already appends the userId for all authenticated request. The frontend doesn't need to pass it

Reply:
cool